website/source/api/agent.html.md

```diff
@@ -490,6 +490,14 @@ state allows its old entries to be removed.
 | ------ | ---------------------------- | -------------------------- |
 | `PUT`  | `/agent/force-leave/:node`   | `application/json`         |
 
+Additionally, by specifying the `prune` flag, a node can be forcibly removed from
+the list of members entirely.
+
+| Method | Path                                    | Produces                   |
+| ------ | --------------------------------------- | -------------------------- |
+| `PUT`  | `/agent/force-leave/:node?prune=true`   | `application/json`         |
```

Comment:
```suggestion
| `PUT`  | `/agent/force-leave/:node?prune`   | `application/json`         |
```
The query parameter doesn't require a value anymore after you accepted one of my earlier suggestions. The `?stale` parameter works similarly and is not documented to accept a parameter value.